### PR TITLE
CIS 6.02: SQL World Readable

### PR DESCRIPTION
**Pull request by charliewolf**

### CODE REVIEW

**Comment by charliewolf** [comment]
@joecheuk No, that policy you linked is a whitelist so we couldn't (without modifying it's behavior quite a bit) support this CIS rule

**Comment by dekuhn** [comment]
@gkowalski-google PTAL. Thanks!

**Comment by gkowalski-google** [comment]
@morgante @AdrienWalkowiak @briantkennedy Can you please review this PR? I made a few updates to Charlies work, I think this is ready to be merged.

**Comment by gkowalski-google** [comment]
@morgante I have updated the constraints to use hyphens here as well.

File: validator/sql_world_readable.rego [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	lib.get_constraint_params(constraint, params)
+
+	asset := input.asset
+	asset.asset_type == "sqladmin.googleapis.com/Instance"
+
+	networks := lib.get_default(asset.resource.data.settings.ipConfiguration, "authorizedNetworks", [])
+	network := networks[_]
+	network.value == "0.0.0.0/0"
+
+	message := sprintf("%v is world readabl", [asset.name])
```
Comment (morgante) [inline validator/sql_world_readable.rego:35]:
```suggestion
	message := sprintf("%v is world readable", [asset.name])
```

File: validator/sql_world_readable_test.rego [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package templates.gcp.GCPSQLWorldReadableConstraintV1
+
+import data.test.fixtures.sql_allowed_authorized_networks.constraints as fixture_constraints
+
+all_violations[violation] {
```
Comment (morgante) [inline validator/sql_world_readable_test.rego:21]: Please switch to use test utils.

**Comment by gkowalski-google** [comment]
@morgante Updated to use test utils and fixed the typo.

**Comment by morgante** [comment]
@gkowalski-google Looks like tests are failing.

**Pull request merged**